src/main/authentication/Security/Authentication/Authenticator.php

```diff
@@ -14,9 +14,9 @@
 use Claroline\AppBundle\Event\StrictDispatcher;
 use Claroline\AppBundle\Persistence\ObjectManager;
 use Claroline\CoreBundle\Entity\User;
-use Claroline\CoreBundle\Event\CatalogEvents\SecurityEvents;
-use Claroline\CoreBundle\Event\Security\UserLoginEvent;
 use Claroline\CoreBundle\Listener\AuthenticationSuccessListener;
+use Claroline\LogBundle\Event\CatalogEvents\SecurityEvents;
+use Claroline\LogBundle\Event\Security\UserLoginEvent;
```

Comment:
Events are not directly related to logs and should stay in the core.

Reply:
Done, thx